feat : 비활성화된 회원 로그인 못하도록 기능 추가

higukang opened the pull request:
# 🚀 Pull Request

로그인과정에 사용자 status 확인 로직 추가

## #️⃣ 연관된 이슈

#180 

## 📋 작업 내용

CustomUserDetailsService.java : 삭제된 회원 로그인 시 예외 발생

higukang committed:
feat : 비활성화된 회원 로그인 못하도록 기능 추가

로그인과정에 사용자 status 확인 로직 추가

CustomUserDetailsService.java : 삭제된 회원 로그인 시 예외 발생

Issues: #180

oowtl commented on module-service/src/main/java/com/checkping/service/member/auth/CustomUserDetailsService.java:
```diff
@@ -25,6 +27,10 @@ public UserDetails loadUserByUsername(String email) throws UsernameNotFoundExcep
         Member member = result.orElseThrow(() -> new UsernameNotFoundException(email));
 
+        if(member.getStatus() == Member.Status.INACTIVE) {
+            throw new BaseException("삭제된 회원입니다", ErrorCode.USER_NOT_FOUND);
```
여기도 커스텀 Exception 을 넣어줄 수 없는 곳인가요?

higukang replied:
커스텀 Exception 적용했습니다!

higukang committed:
chore : 삭제된 회원으로 로그인할 시 커스텀 예외 추가

CustomUserDetailsService.java : 삭제된 회원으로 로그인할 시 커스텀 예외 추가

InactiveMemberException.java : 삭제된 회원으로 로그인할 시 커스텀 예외

Issues: #180
